Jenkinsfile_CNP

```diff
@@ -136,7 +136,7 @@ withPipeline(type, product, component) {
   }
 
   before('functionalTest:aat') {
-    env.SKIP_FUNCTIONAL_TESTS = 'true'
+    env.TESTS_SELECTOR="@smoke-tests"
```

Comment:
Would this not just be doing the same as the smoke test step? It goes deploy aat -> smoke aat -> functional aat right?

Reply:
yes, it will do the same what smoke test. We just need to run anything there, agreed with Adam and Robin that we run there smoke test for now

Reply:
do we want to have an aat tag which we can then add to the tests which we can then use in the future?

Reply:
we can add it when there will be need, for now we dont even know if we will be running e2e or api tests. It is to have something there to not be marked in red in program reports